### PR TITLE
512 gallery manager

### PR DESCRIPTION
**Pull request by joncppl**

### CODE REVIEW

File: dispatch/static/manager/src/js/actions/GalleryActions.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as types from '../constants/ActionTypes'
```
Comment (joncppl) [inline dispatch/static/manager/src/js/actions/GalleryActions.js]: duplicate file?

File: dispatch/static/manager/src/js/api/dispatch.js [hunk 2]
```diff
@@ -317,6 +317,22 @@ const DispatchAPI = {
...
+    },
+    create: (token, data) => {
+      return postRequest('galleries', null, data, token)
+    },
+    save: (token, galleryId, data) => {
+      return patchRequest('galleries', galleryId, data, token)
+    },
+    delete: (token, galleryId) => {
+      return deleteRequest('galleries', galleryId, null, token)
+  },
```
Comment (psiemens) [inline dispatch/static/manager/src/js/api/dispatch.js]: Missing closing bracket here

File: dispatch/static/manager/src/js/components/GalleryEditor/DnDThumb.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    spanStyle.width = width
+  }
+  if (height) {
+    spanStyle.height = height
+  }
+
+  return connectDragSource(
+    <div
+      style={spanStyle}
+      >
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/GalleryEditor/DnDThumb.js]: Put the above three lines on a single line

File: dispatch/static/manager/src/js/pages/Galleries/GalleryIndexPage.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      }
+    },
+    searchListItems: (query) => {
+      dispatch(galleriesActions.search(query))
+    }
+  }
+}
+
+
+function TagsPageComponent(props) {
```
Comment (psiemens) [inline dispatch/static/manager/src/js/pages/Galleries/GalleryIndexPage.js]: Rename this to GalleriesPage

File: dispatch/static/manager/src/js/pages/Galleries/GalleryIndexPage.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  return (
+    <ItemIndexPage
+      typeSingular='gallery'
+      typePlural='galleries'
+      displayColumn='title'
+      {... props} />
+  )
+}
+
+const TagsPage = connect(
```
Comment (psiemens) [inline dispatch/static/manager/src/js/pages/Galleries/GalleryIndexPage.js]: Same here

File: dispatch/static/manager/src/styles/components/imagegallery_editor.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
+@import '../utilities/_colors.scss';
+
+.c-imagegallery-editor-heading {
```
Comment (psiemens) [inline dispatch/static/manager/src/styles/components/imagegallery_editor.scss]: Let's use `c-gallery-` instead of `c-imagegallery`

**Review by psiemens — CHANGES_REQUESTED** [review]

**Comment by joncppl** [comment]
included the embed in this pr as well

**Pull request merged**